src/api/app/components/buildresult_status_link_component.rb

```diff
@@ -0,0 +1,27 @@
+class BuildresultStatusLinkComponent < ApplicationComponent
+  def initialize(repository:, architecture:, project:, package_name:, build_status:, build_details:)
+    super
+
+    @repository = repository
```

Comment:
Are `repository`, architecture`, `project` and `package_name` all names? If so, they should all end with `_name` to be consistent. It would be clearer what is expected to be passed into this view component.

Reply:
right those are not objects, I changed the naming @dmarcoux